packages/calypso-analytics/src/tracks.ts

```diff
@@ -215,13 +215,13 @@ export function recordTracksEvent( eventName: string, eventProperties?: any ) {
 		if (
 			! /^calypso(?:_[a-z0-9]+){2,}$/.test( eventName ) &&
 			! /^jetpack(?:_[a-z0-9]+){2,}$/.test( eventName ) &&
+			! /^remotedatablocks(?:_[a-z0-9]+){2,}$/.test( eventName ) &&
 			! /^wpcom_dsp_widget(?:_[a-z0-9]+){2,}$/.test( eventName ) &&
```

Comment:
Not a blocker for this PR, but a nice potential follow-up: would be nice if we avoid this regex repetition here by having an array of allowed source prefixes declared and checking against that.

Reply:
Fixed in [786e273](https://github.com/Automattic/wp-calypso/pull/96007/commits/786e2736bd2d662a8b17a6fcbefce2ddfeb8f7b7). Please have a look.